x-pack/plugins/lens/public/datatable_visualization/expression.tsx

```diff
@@ -30,6 +35,12 @@ export interface DatatableProps {
   args: Args;
 }
 
+type DatatableRenderProps = DatatableProps & {
+  formatFactory: FormatFactory;
+  executeTriggerActions: UiActionsStart['executeTriggerActions'];
+  getType: Function;
```

Comment:
Can you make this type stronger? it's not just a function, it's a function that returns an AggType?